src/content/public goods funding.md

```diff
@@ -0,0 +1,281 @@
+
+ii. Other donors who wanted `park B` were able to express the strength of their preferences for that public good by contributing more money to its development. Perhaps, unknown to most people, `park B` has more long-term benefits for the community than `park A`. In this scenario, quadratic voting/funding prevents widespread ignorance from canceling out the wisdom of informed citizens. 
+
+#### Reducing negative externalities
+
+Under a regular free-market democratic system, everyone would be forced to pay equally for a public good—whether they wanted it or not. But, as demonstrated in the example, quadratic voting/funding ensures individual pay for a public if and only if they want it. 
+
+Another negative externality that quadratic funding mitigates is the incentive to maximize profits at the expense of consumers (the development of `Park B` was still subsidized by the matching pool even if it had low support). Why is this important? It means those who build public goods have fewer reasons to try and extract profits from consumers through any means possible. If `park B` were built solely with investors' funds, the incentive to recoup the initial investment—for example, by charging visitors high fees—would be high. 
+
+## How Ethereum improves quadratic funding for public goods {#how-ethereum-improves-quadratic-funding} 
+
+Some may ask: "Why do you need a blockchain to coordinate a quadratic funding round when it can be implemented on a regular online platform?" There are two major issues with running quadratic voting on regular applications:
+
+1. **Sybil attacks**: A [Sybil attack](https://academy.binance.com/en/articles/sybil-attacks-explained) occurs when someone participating in an online platform creates multiple identities to increase their influence within that system. Deploying social media bots is an example of a Sybil attack. Due to the high value of individual contributions in a quadratic funding scheme, cheaters may try to game the system by creating different profiles and swing voting outcomes. This would corrupt the system and reduce the usefulness of quadratic voting in gauging interest in specific public goods. 
+
+2. **Bribery and collusion**: It is possible to bribe voters to donate to a particular project (even if they prefer another option), thereby artificially inflating perceived preference for the former. This defeats the purpose of voting since the votes cast by a bribed individual may not reflect their true preference. Hence, just like Sybil attacks, bribery and collusion reduce the effectiveness of quadratic funding mechanisms. 
+
+Ethereum solves both problems in the following ways:
```

Comment:
I'm not sure this problem has been **solved** - it is still definitely an area of rapid development, but the arms race between Sybil attackers and defenders is real.